qiskit/pulse/commands/sample_pulse.py

```diff
@@ -104,27 +105,34 @@ def draw(self, dt: float = 1,
              style: Optional['PulseStyle'] = None,
              filename: Optional[str] = None,
              interp_method: Optional[Callable] = None,
-             scaling: float = 1, interactive: bool = False):
+             scale: float = 1, interactive: bool = False,
+             scaling: Optional[float] = 1):
```

Comment:
```suggestion
             scaling: float = 1):
```
`scaling` must still be a float.